tdrs-backend/tdpservice/parsers/case_consistency_validator.py

```diff
@@ -517,7 +533,8 @@ def __validate_t5_atd_and_ssi(self):
                     msg=(
                         f"{t5_model_name} People in territories must have value = 2:No for "
                         f"{self.__get_error_context('REC_SSI', schema)}."
-                    )
+                    ),
+                    deprecated=True
```

Comment:
confirmed that the following error messages deprecated for territories ✔️ 

`T5 Adults in territories must have a valid value for Item 19C (Received Disability Benefits: Permanently and Totally Disabled).`
`T5 People in territories must have value = 2:No for Item 19E (Received Disability Benefits: SSI).`